### ISSUE TITLE
installation errors

### ISSUE THREAD

**Issue by datakid** [broadinstitute/deepometry#19]
FYI, in a clean virtualenv, I get this error when following the installation instructions:

```
<guff>
...
<guff>
RuntimeError: Numpy must be installed before installing javabridge.

It cannot be installed automatically when pip-installing javabridge.

See https://github.com/CellProfiler/python-javabridge/issues/30

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /config/binaries/deepometry/0.0.1/build/javabridge
Storing complete log in /root/.pip/pip.log
```

After installing numpy (and upgrading pip) I needed to install <java>-devel (bc I'm on CentOS - Ubuntu users probably need something called <java>-dev)

**Comment by mcquin** [comment]
Thanks for the report! It's a known issue that javabridge (required for parsing CIFs) requires numpy. I'll update the installation instructions in the README.md to explicitly install numpy before installing the deepometry dependencies.

**Comment by mcquin** [comment]
Closed via addf97a3ff5ec40f3c02c56b56c5f04936525a41
